pkg/ccl/logictestccl/testdata/logic_test/multi_region

```diff
@@ -108,9 +108,9 @@ JOIN system.namespace parent ON (e."parentID" = parent.id)
 WHERE e.name='crdb_internal_region'
 ----
 parentSchemaID  db_name                                       enum_name
-112             multi_region_test_db                          crdb_internal_region
-116             multi_region_test_explicit_primary_region_db  crdb_internal_region
-108             region_test_db                                crdb_internal_region
+111             multi_region_test_db                          crdb_internal_region
```

Comment:
I don't understand why these IDs are off by 1. (same to the following test data changes under `schemachangerccl`)

Reply:
When creating the database, we also create the public schema and the multi-region enum. These creations now happen in the different sequence than before.